wsclient/src/main/java/org/hpccsystems/ws/client/platform/WorkunitInfo.java

```diff
@@ -382,26 +523,49 @@ public WorkunitInfo setOriginalEclWatchUrl(String serverhost) {
 	    return applicationValues;
 	}
 	
-	public ECLException[] getRawExceptions()
+	public org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLException[] getRawExceptions()
```

Comment:
eclexceptioninfo or hopefully eclexceptionwrapper?

Reply:
this is getRawExceptions, which translates the wrapper exceptions back into raw exceptions